File: apps/ios/Barns/BarnsApp/Domain/Entities/Notice.swift
import Foundation

struct Notice: Equatable, Identifiable, Sendable {
    let id: String
    let title: String
    let body: String
    let publishedAt: Date
    let priority: NoticePriority
    let imageUrl: URL?
}

enum NoticePriority: String, Equatable, Sendable {
    case low
    case normal
    case high
}

File: apps/android/app/src/main/java/com/barns/app/domain/model/Notice.kt
package com.barns.app.domain.model

import java.time.Instant

data class Notice(
    val id: String,
    val title: String,
    val body: String,
    val publishedAt: Instant,
    val priority: NoticePriority,
    val imageUrl: String?,
)

enum class NoticePriority { LOW, NORMAL, HIGH }
